star-ros2/src/star_spi_bridge/src/spi_driver.cpp

```
@@ -39,88 +53,116 @@ struct spi_ioc_transfer
-inline void set_spi_xfer_bits_per_word(spi_ioc_transfer & xfer, uint8_t value)
-{
+inline void set_spi_xfer_bits_per_word(spi_ioc_transfer &xfer, uint8_t value) {
 #ifdef __linux__
   xfer.bits_per_word = value;
 #else
   xfer.bits_per_word_ = value;
 #endif
 }
-}  // namespace
+
+inline bool is_valid_frame_type(FrameType frame_type) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)